healthcheck/utils.py

```diff
@@ -0,0 +1,26 @@
+import errno
+import os
+
+
+def file_exists(path):
+    """Return True if a file exists at `path` (even if it can't be read),
+    otherwise False.
+
+    This is different from os.path.isfile and os.path.exists which return
+    False if a file exists but the user doesn't have permission to read it.
```

Comment:
Sorry, I hadn't properly understood the permissions issues at hand with directories and so on. Will make it return a more sensible result.